src/components/Register.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import '../assets/styles/register.scss'
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import UserService from '../services/UserService';
+
+const SignupSchema = Yup.object().shape({
```

Comment:
You can create a separate file for each component's validation. For example, we can make a `Register` folder inside components which will contain `Register.js` (component's file), a separate file containing and exporting validation schema (`validations.js`), and `index.js` (loadable for component file).